<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>362</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GUI 프로그램 만들기</string>
  </property>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>이름</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_phone">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>전번</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ed_phone">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="ed_memo">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>111</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lb_memo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>메모</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>저장</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_del">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>190</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>삭제</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_png">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>210</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../../../Users/jjkkh/Pictures/Screenshots/knulogo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cam">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>촬영</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_video">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>20</y>
     <width>161</width>
     <height>161</height>
    </rect>
   </property>
   <property name="text">
    <string>video</string>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>220</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
